impls/msft-eshoppen.yaml

```diff
@@ -0,0 +1,96 @@
+name: msft-eshoppen
+description: https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/msft-eShoppen.md
+tags:
+  kind: web
+  complexity: moderate
+  category: cloud
+config:
+  pipeline:
+    calculation: msft-eshoppen
+    normalization: default
+    aggregation: default
+    enrichment: default
+graph:
+  front-end:
+    config:
+      embodied_c: 350 # kgCO2eq
+      reserved_time: 1 # hour
+      resources_reserved: 1
+      total_resources_available: 1
+      lifespan: 4 #years
+    children:
+      cpu:
+        observations:
+          common:
+            server:  Intel® Xeon® Platinum 8272CL
+            tdp: 28 # W
+            carbon_intensity: 951
+            tdp_coeff: 0.32
```

Comment:
I see how this works now, it's pulling the TD coefficient using the utilization. We should represent this as a "teads" or some sort of generic "lookup" model, will discuss more in an issue.